README.md

```diff
@@ -19,6 +19,7 @@ Add the plugin, reporter and reporter configuration in your `karma.conf.js`.
   reporters: ['progress', 'karma-remap-istanbul'],
   remapIstanbulReporter: {
     remapOptions: {}, //additional remap options
+    writeReportOptions: {}, //additional writeReport options
```

Comment:
Can you rename it from `writeReportOptions` to `reportOptions` as this is what remap-istanbul calls the arg in their docs: https://github.com/SitePen/remap-istanbul/blob/master/src/writeReport.js#L22